packages/stream_video/lib/src/call_state.dart

```diff
@@ -19,7 +19,7 @@ class CallState extends Equatable {
       preferences: preferences,
       currentUserId: currentUserId,
       callCid: callCid,
-      createdByUserId: '',
```

Comment:
Did you mix this PR with another fix by accident?

Reply:
I also replaced the `createdByUserId` with `createdByUser` in this PR to solve two gaps that Martin reported we had in CallState.